shell/browser/native_browser_view_views.cc

```diff
@@ -137,6 +137,13 @@ gfx::Rect NativeBrowserViewViews::GetBounds() {
   return iwc_view->GetView()->bounds();
 }
 
+void NativeBrowserViewViews::RenderViewReady() {
+  auto* iwc_view = GetInspectableWebContentsView();
```

Comment:
nit: say the type. It wasn't totally obvious to me that this returns an `InspectableWebContentsView*`, so I think this helps readability ein Bißchen

```suggestion
  InspectableWebContentsView* iwc_view = GetInspectableWebContentsView();
```